src/IsoCodes/PhoneNumber.php

```diff
@@ -25,7 +26,11 @@ public static function validate($phoneNumber, $country)
         }
         $country = strtoupper($country);
         $phoneUtil = PhoneNumberUtil::getInstance();
-        $numberProto = $phoneUtil->parse($phoneNumber, $country);
+        try {
+            $numberProto = $phoneUtil->parse($phoneNumber, $country);
+        } catch (NumberParseException $e) {
```

Comment:
Please handle `\Exception` instead, since the [libphonenumber](https://github.com/giggsey/libphonenumber-for-php) project may add some more custom exceptions alongside the `NumberParseException` class